src/pages/Games.tsx

```diff
@@ -1,36 +1,43 @@
-import Form from "../components/Games/Form";
-import Cards from "../components/Games/Cards";
 import { useState } from "react";
-
-import { Container, Box } from "@mui/material";
+import { Box, Container } from "@mui/material";
 import SportsEsportsIcon from "@mui/icons-material/SportsEsports";
 import Title from "../components/Title";
 import StyleIcon from "@mui/icons-material/Style";
 import { useTheme } from "@mui/material/styles";
 
-import { Card } from "../components/Games/Cards";
+import Form from "../components/Games/Form";
+import Cards, { Card } from "../components/Games/Cards";
 
 export default function App() {
   const theme = useTheme();
   const [cards, setCards] = useState<Card[]>([]);
+  const [editingCard, setEditingCard] = useState<Card | null>(null);
+
   const handleFormSubmit = (newCard: Card) => {
-    setCards((prevCards) => [...prevCards, newCard]);
+    if (editingCard) {
+      const updatedCards = cards.map((card) =>
+        card.id === editingCard.id ? newCard : card
+      );
+      setCards(updatedCards);
+      setEditingCard(null);
+    } else {
+      setCards((prevCards) => [...prevCards, newCard]);
+    }
   };
 
   const deleteCard = (id: string) => {
-    setCards((prevCards) => {
-      const filteredCards = prevCards.filter((card: Card) => card.id !== id);
-      return filteredCards;
-    });
+    setCards((prevCards) => prevCards.filter((card: Card) => card.id !== id));
   };
+
   return (
     <Box
       sx={{
         flexDirection: { xs: "column", sm: "column", md: "row" },
         display: "flex",
         justifyContent: "space-between",
         bgcolor: theme.palette.background.default,
-        width: "100vw",
+        width: "98.9vw",
```

Comment:
values like this look a bit suspicious, maybe it is better way  to handle this?

Reply:
I know that it is bad but I couldn't do it on the full size of the screen

Reply:
please mark this as `// TODO`